src/node/context.h

```diff
@@ -36,6 +37,7 @@ class WalletClient;
 struct NodeContext {
     std::unique_ptr<CConnman> connman;
     std::unique_ptr<CTxMemPool> mempool;
+    std::shared_ptr<CBlockPolicyEstimator> feeEstimator;
```

Comment:
Why does this need to be shared? `NodeContext` should have unique ownership of the feeEstimator object.

Reply:
Right, fixed